lib/rucio/core/credential.py

```diff
@@ -144,11 +143,20 @@ def get_signed_url(rse_id, service, operation, url, lifetime=600):
             s3op = 'delete_object'
 
         with METRICS.timer('signs3'):
-            s3 = boto3.client('s3', endpoint_url='https://' + host + ':' + port, aws_access_key_id=access_key, aws_secret_access_key=secret_key, config=Config(signature_version=signature_version, region_name=region_name))
-
-            signed_url = s3.generate_presigned_url(s3op, Params={'Bucket': bucket, 'Key': key}, ExpiresIn=lifetime)
-
-    elif service == 'swift':
+            s3 = boto3.client(
+                's3', endpoint_url=f'https://{host}:{port}',
+                aws_access_key_id=access_key, aws_secret_access_key=secret_key,
+                config=Config(
+                    signature_version=signature_version,
+                    region_name=region_name
+                )
```

Comment:
I'm not so much in favour of all these linebreak changes... especially ones that result in empty line parentheses.